.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
That's weird, I was almost sure the indentation was using tabs from the beginning because I normally prefer spaces as well. :hushed: I'll fix the `.editorconfig` and adjust the indentation then.
I'm using @atom as my "IDE" (text editor) by the way.